libraries/ms-common/inc/stm32f0xx/gpio_cfg.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#define GPIO_CFG_NUM_PINS_PER_PORT 16
+#define GPIO_CFG_NUM_PORTS 6
+
+#define GPIO_CFG_TOTAL_PINS (GPIO_CFG_NUM_PINS_PER_PORT * GPIO_CFG_NUM_PORTS)
```

Comment:
You want to wrap all macro uses (i.e. `((GPIO_CFG_NUM_PINS_PER_PORT) * (GPIO_CFG_NUM_PORTS))` in case one of them expands to an expression. For example, if you had
````
#define VAL_A 4 + 4
#define VAL_B VAL_A * 3
#define VAL_C ((VAL_A) * 3)
````
`VAL_B` wouldn't have the expected value, but `VAL_C`